riotfile.py

```diff
@@ -1434,7 +1434,8 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             name="mongoengine",
             command="pytest {cmdargs} tests/contrib/mongoengine",
             pkgs={
-                "pymongo": latest,
```

Comment:
[Line 749] Should we be testing the specific versions you have if/else for in the patch code? Specifically 4.5 (maybe 4.9 but I guess that's a bit overkill with `latest` already there)

Reply:
We already do this for pymongo. Our mongoengine essentially just enables our pymongo integration. It doesn't do anything special. 

I am actually planning to remove the mongoengine integration in v3.0.0